01week/datatypes.js

```diff
@@ -47,6 +58,12 @@ var u = undefined;
 var p = 1;
 var a = "b";
 
+document.getElementById("bool").innerHTML = typeof f;
+document.getElementById("null").innerHTML = typeof c;
+document.getElementById("undef").innerHTML = typeof u;
+document.getElementById("num").innerHTML = typeof p;
+document.getElementById("notnum").innerHTML = isNaN(p);
+document.getElementById("str").innerHTML = typeof a;
 
```

Comment:
I like that you are using HTML to show your data on the page, but it wasn't necessary for this project.